prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -1204,6 +1204,9 @@ struct rxr_op_entry *rxr_pkt_get_tagrtm_rx_entry(struct rxr_ep *ep,
 			efa_eq_write_error(&ep->util_ep, FI_ENOBUFS, FI_EFA_ERR_RX_ENTRIES_EXHAUSTED);
 			return NULL;
 		}
+		rxr_tracing(msg_recv_unexpected, rx_entry->msg_id,
```

Comment:
The event name is a little ambiguous. It should have `tagged` in it to indiciate the message is ad tagged message. Probably something like `uexpected_tagged_message_received`

Reply:
Can you also add an trace in `rxr_pkt_get_msgrtm_rx_entry` for non-tagged version of this event.

Reply:
I named the events msg_recv_unexpected_{tagged/untagged} to match the style of the other events.